client-side-encryption.md

```diff
@@ -0,0 +1,856 @@
+  secret: STYTCH_SECRET,
+})
+
+const emailResponse = await prompts({
+  type: 'text',
+  name: 'email',
+  message: 'Enter your email address',
+})
+
+const stytchResponse = await client.otps.email.loginOrCreate({
+  email: emailResponse.email,
+})
+
+const otpResponse = await prompts({
+  type: 'text',
+  name: 'code',
+  message: 'Enter the code sent to your email:',
+})
+
+const authResponse = await client.otps.authenticate({
+  method_id: stytchResponse.email_id,
+  code: otpResponse.code,
+  session_duration_minutes: 60 * 24 * 7,
+})
+
+let sessionResp = await client.sessions.get({
+  user_id: authResponse.user_id,
+})
+
+// Mint PKPs via Lit Relay
+const litRelay = new LitRelay({
+  relayUrl: LitRelay.getRelayUrl(LIT_NETWORK.DatilDev),
+  relayApiKey,
+})
+
+// Create the auth method
+const authMethod = {
+  authMethodType: AuthMethodType.StytchEmailFactorOtp,
+  accessToken: sessionResp.session_token,
+}
+
+const mintResult = await litRelay.mintPKPWithAuthMethods([authMethod], {
+  pkpPermissionScopes: [[1]], // This scope allows signing any data
+})
+
+console.log('PKP minting successful:', {
+  tokenId: mintResult.pkpTokenId,
+  publicKey: mintResult.pkpPublicKey,
+  ethAddress: mintResult.pkpEthAddress,
+})
+```
+
+### 2.3 Payment Delegation System
+
+Implement Lit Protocol's Payment Delegation Database API to enable Storacha to act as the payer for all user Lit transactions. This removes the burden of capacity credit management from individual users and simplifies onboarding.
+
+#### 2.3.1 Payment Delegation Database Architecture
+
+The [Lit Protocol Payment Delegation Database](https://developer.litprotocol.com/paying-for-lit/payment-delegation-db) enables applications to pay for user operations without requiring users to hold capacity credits.
+
+> The Payment Delegation Database is a smart contract deployed on Lit's rollup, Chronicle Yellowstone. Lit's Relayer server has been updated to provide two new API routes to interface with the Payment Delegation Database contract:
+
+> **POST /register-payer**: This route is used to register a new payer and will have a Capacity Credit minted for it which can be delegated to payees to pay for their usage of Lit
+
+> **POST /add-users**: This route is used to add users (as Ethereum addresses) as payees for a specific payer. This allows the payer to pay for the usage of Lit for each user, without each user having to own a Capacity Credit
+
+> Reference: https://developer.litprotocol.com/paying-for-lit/payment-delegation-db#the-payment-delegation-database
+
+The system works through a allowlist-based approach:
+
+- **Storacha as Payer**
+
+  - **Registration**: Storacha registers as a payer with its own PKP wallet and capacity credits
+  - **Key Management**: Maintains a secure master payer secret key for delegation database operations
+  - **Centralized Funding**: Funds capacity credits for all user operations through a single account, eliminating individual user payment complexity
```

Comment:
I understand that we’re going to sponsor their usage, but I was wondering, beyond the plans that include access to private data, do we have a defined quota for capacity credits included in each plan? And will there be an extra charge if the user exceeds that limit?
Basically, what’s the maximum amount of capacity credits we’re allowing per plan?